webdriver/tests/take_element_screenshot/user_prompts.py

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
I would suggest to add a `# META: timeout=long` in the first line, as what we have for all the other user prompt tests. Reason is that those tests can take a bit longer, and would cause timeouts.

Reply:
Done.

Reply:
I cannot see this entry. Do I miss something?

Reply:
![image](https://user-images.githubusercontent.com/1809089/65711692-4cab3080-e04a-11e9-8278-faa07f1e86f6.png)

 I definitely see it there, but I had to click an icon to expand a collapsed section.

Reply:
Not that easy to see. Thanks for highlighting that. Expanding this comment section brings-up those meta tags.